frontend/src/test/scala/bloop/bsp/BspMetalsClientSpec.scala

```diff
@@ -167,7 +167,7 @@ class BspMetalsClientSpec(
         "-P:semanticdb:failures:warning",
         "-P:semanticdb:synthetics:on",
         "-Xplugin-require:semanticdb",
-        s"-Xplugin:path-to-plugin/semanticdb-scalac_2.12.15-4.4.30.jar.jar",
+        s"-Xplugin:path-to-plugin/semanticdb-scalac_2.12.17-4.4.30.jar.jar",
```

Comment:
```suggestion
        s"-Xplugin:path-to-plugin/semanticdb-scalac_2.12.17-4.6.0.jar.jar",
```

Reply:
Thanks, I looked again and fixed 2 more places. Although, I think they we only testing if the correct settings are added, which is why they didn't pop up in tests probably.